process_report/processors/new_pi_credit_processor.py

```diff
@@ -0,0 +1,190 @@
+import sys
+
+from dataclasses import dataclass
+import pandas
+import pyarrow
+
+from process_report import util
+from process_report.invoices import invoice
+from process_report.processors import discount_processor
+
+
+@dataclass
```

Comment:
Why is this a dataclass? It doesn't appear to have any class attributes. I guess the same question applies to `discount_process.DiscountProcessor` as well.

Reply:
`NewPICreditProcessor` does have `old_pi_filepath` and `limit_new_pi_credit_to_partners` as its class attributes, but as for `DiscountProcessor`, you are right, it has none. I guess I made them dataclasses for uniformity with all the other processors and invoices and didn't put too much thought into it. 

I will remove `@dataclass` from `DiscountProcessor`